.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Notify for frontend
+/frontend/ @Slessi
```

Comment:
we could also create a github team called `frontend` and add it here as `@siderolabs/frontend`